src/main/java/core/basesyntax/FootballMatch.java

```diff
@@ -15,6 +15,8 @@
      * если не угадал ничего - возвращать 0.</p>
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+        return result1 == bet1 && result2 == bet2 ? 2 :
+                  result1 > result2 && bet1 > bet2 || result2 > result1 && bet2 > bet1 ? 1 :
```

Comment:
Code Style
Please, view this article: [https://mate-academy.github.io/style-guides/java/java.html](url)